<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Partner</class>
 <widget class="QMainWindow" name="Partner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Партнёры</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/Мастер пол.ico</normaloff>images/Мастер пол.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #123456;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tablePartner">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>700</width>
      <height>241</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;
max-width: 700px;
background-color:#666000;
border-radius: 15px;</string>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>№</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Тип</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Название</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Директор</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Почта</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Телефон</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Адрес</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ИНН</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Рейтинг</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="addNewPartnerButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:#666000;

border: 6px;
color:black;
font: 87 10pt \&quot;Segoe UI Black\&quot;;\n\;
}
QPushButton:hover {
background-color:rgb(91, 91, 91);
}
QPushButton:pressed {
background-color: rgb(61, 61, 61);
}

</string>
    </property>
    <property name="text">
     <string>Добавить нового партнёра</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
